Library/Formula/kibana.rb

```diff
@@ -0,0 +1,64 @@
+class Kibana < Formula
+  desc "Visualization tool for elasticsearch"
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana/archive/v4.0.2.tar.gz"
+  sha256 "facb5b2eb31de38ebf6e30bb656c62a512e540623df98b33655f5b0da896dd27"
+  head "https://github.com/elastic/kibana.git"
+
+  depends_on "node"
+
+  def install
+    ENV["HOME"] = buildpath/"home"
```

Comment:
Team had a discussion where `home` was deemed too generic a name for this variable - It'll be changed elsewhere soon as well. Can you set it as `buildpath/".brew_home"`. Sorry for the crossed wires!